META.md

```diff
@@ -10,19 +10,22 @@
   - [Increment a Version in Every Plugin](#increment-a-version-in-every-plugin)
     - [Increment Version in OpenSearch](#increment-version-in-opensearch)
     - [Create a 1.2.3 Manifest](#create-a-123-manifest)
+    - [Prepare for Increments](#prepare-for-increments)
     - [Increment Version in Plugins](#increment-version-in-plugins)
     - [Commit and Push Changes](#commit-and-push-changes)
     - [Create Pull Requests](#create-pull-requests)
-      - [common-utils and job-scheduler](#common-utils-and-job-scheduler)
-      - [alerting](#alerting)
+      - [common-utils](#common-utils)
+      - [job-scheduler](#job-scheduler)
       - [min-SNAPSHOT](#min-snapshot)
+      - [alerting](#alerting)
       - [Remaining Plugins](#remaining-plugins)
       - [Update job-scheduler Snapshots](#update-job-scheduler-snapshots)
     - [Update the Manifest](#update-the-manifest)
+  - [Backporting in OpenSearch Plugins](#backporting-in-opensearch-plugins)
```

Comment:
I couldn't find a better place, but Meta.md doesnt feel like the right place to have the backport workflows.
May be https://github.com/opensearch-project/opensearch-plugins/blob/main/STANDARDS.md?

Reply:
Yeah you are right. It doesn't look right in `META.md`. What do you think between `STANDARDS.md` and `CONTRIBUTING.md`?